src/main/java/com/piramalswasthya/db/config/DatasourceConfig.java

```diff
@@ -0,0 +1,49 @@
+package com.piramalswasthya.db.config;
+
+import javax.sql.DataSource;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.jdbc.DataSourceBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+@Configuration
+public class DatasourceConfig {
+	
+	@Value("${spring.datasource.dbiemr.url}")
+    private String dbiemrurl;
+	
+	@Value("${spring.datasource.dbidentity.url}")
+    private String dbidentityurl;
+	
+	@Value("${spring.datasource.dbreporting.url}")
+    private String dbreportingurl;
+	
+	@Value("${spring.datasource.db1097identity.url}")
+    private String db1097identityurl;
+	
+	@Bean
+	@ConfigurationProperties(prefix = "spring.datasource.dbiemr")
+	public DataSource datasource1() {
```

Comment:
Can we use better variable names instead of 1,2,3...
Applies for other places too.